policykit/policyengine/tests/test_models.py

```diff
@@ -134,7 +134,8 @@ def setUp(self):
         self.data_store = DataStore.objects.create(
```

Comment:
This test file doesn't work for me locally it fails with errors about Permissions missing. I guess it doesn't get run with `python manage.py test` for some reason. Do you use this locally @rickygv99 or is it just old?